content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Results of the GRASS GIS student grants - Part 1"
+date: 2022-05-20T21:12:52+02:00
+layout: "news"
+author: Linda Kladivova and the GRASS Development Team
+---
+
+## Linda's path in her own words
+### From enhancing first time user experience to Single-Window in GRASS GIS
+
+<a href="/images/gallery/gui/grass8_firststartup.png">
+  <img src="/images/gallery/gui/grass8_firststartup.png"
+   alt="GRASS GIS at first launch"
+   title="GRASS GIS at first launch"
+   width="70%" style="float:right;padding-left:20px;padding-top:20px">
+</a>
+
+I started contributing to GRASS GIS in the summer of 2020 as part 
+of the Google Summer of Code (GSoC) project
+[*"Creation of a new GRASS GIS startup mechanism"*](https://trac.osgeo.org/grass/wiki/GSoC/2020/StartupWindow), 
+which I applied for as a complete programming illiterate. Although
+I felt a bit clumsy at first, with the great help of my mentors, I 
+managed to **significantly improve the GRASS data catalog**. 
+Looking back now, this was the starting point for all the
+upcoming fundamental changes in the graphical user interface (GUI).
+
+In the following months, as part of my master thesis, I focused on 
+the topic of a **new startup mechanism** and **the improvement 
+of the first-time user experience**. The whole effort led to
+eliminating the hard-to-grasp startup screen and implementing
+a **new first-time user mode**, which made GRASS significantly more
+accessible to new users.
+
+<a href="/images/gallery/gui/grass8_single_window_layout_mockup.png">
+  <img src="/images/gallery/gui/grass8_single_window_layout_mockup.png"
+   alt="Mock-up for GSoC 2021 proposal"
+   title="Mock-up for GSoC 2021 proposal"
+   width="60%" style="float:left;padding-right:20px;padding-top:20px">
+</a>
+
+What helped a lot in steering the further implementations were the 
+online community surveys I created as part of my master thesis. 
+Thanks to them, other thorny topics emerged among which I was mostly 
+interested in the need to unify GRASS into a **Single-Window layout**. 
+From the very beginning, I wanted to make working with GRASS as pleasant
+as possible, but until then I have largely focused on new users. 
```

Comment:
Maybe "had largely focused?" I am not sure about the right grammar...

Reply:
I agree

Reply:
done! thanks for catching it :)